lib/tasks/traject.rake

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'traject'
+require './lib/psulib_traject/indexer'
+require 'sidekiq'
```

Comment:
this should just be `require 'psulib_traject'`

Reply:
when changing this to `psulib_traject` it can't find it, when changing it to `./lib/psulib_traject` it finds it, but nothing within the modules require block loads. i.e `LoadError: no such file to load -- psulib_traject/hathi_overlap_reducer` I'm sure it's something i'm doing wrong with pathing or something, this is when running `bundle exec rake -T`

Reply:
I pushed up a commit to fix this. It goes in the Rakefile.